webapp/client/cypress/integration/FahrtkostenpauschalePersonA.spec.js

```diff
@@ -6,6 +6,7 @@ describe("FahrtkostenpauschalePersonA", () => {
   context("with no data", () => {
     beforeEach(() => {
       cy.visit("/lotse/step/person_a_requests_fahrtkostenpauschale");
+      cy.extended_footer_is_disabled(true);
```

Comment:
Why do you test this for all calls to cy.visit?

Couldn't we just test it as a unit test for the different pages that when we set the correct attribute that we then also set no footer?
However, we might need to test in the backend that we set the correct flag.

I am fine either way but I was just a little bit confused that we test this part so much while we know that it only depends on the flag.

Reply:
I had this thought as well after I changed all the existing tests and you are actually right. We added a new functionality --> disabling the "extended" footer and therefore we should create a new test that checks all that pages without the extended footer and also the other pages which retain it. I will change it